source/quick-start.txt

```diff
@@ -39,32 +85,21 @@ Use the ``dotnet add`` command to add the {+driver-short+} to your project as a
 
    dotnet add package MongoDB.Driver
 
-Create a MongoDB Cluster
-------------------------
-
-.. include:: /includes/quick-start/atlas-setup.rst
-
 Query Your MongoDB Cluster from Your Application
 ------------------------------------------------
 
-First, save your Atlas connection string to an environment variable named ``MONGODB_URI``. This prevents potential security issues caused by including your credentials in the source code.
+In this step, you'll use the {+driver-short+} 
+to connect to your MongoDB cluster and run a query on the sample data.
 
-.. code-block:: bash
-
-   export MONGODB_URI='<your atlas connection string>'
-
-.. important::
-
-   Make sure to replace the "<username>" and "<password>" sections of the connection string with the username and password of your Atlas user.
-
-Next, open the file named ``Program.cs`` in the base directory of your project. Copy the
+Open the file named ``Program.cs`` in the base directory of your project. Copy the
 following sample code into ``Program.cs`` 
 
 .. literalinclude:: /includes/quick-start/Program.cs
    :language: csharp
    :dedent:
 
-The sample code runs a query against your sample dataset in MongoDB Atlas. Run the sample code with the following command from your command line:
+The sample code runs a query against your sample dataset in MongoDB Atlas. Run the 
+sample code with the following command from your command line:
```

Comment:
Suggestion:

1) I prefer using "this" over "the" when referring to an instance of something introduced just prior because it's more specific.
2) Sometimes there's a fine line between possessive "your" and non-possessive "the".
3) Reorder the final sentence so "command" is closest to the command itself which can be helpful for understanding its purpose.

```suggestion
This sample code runs a query against your sample dataset in MongoDB Atlas. Run it
from your command line using the following command:
```

Reply:
Changed "using" to "by using" as required by Style Guide, but adopted all other suggested changes